packages/networks/src/defaults.ts

```diff
@@ -14,6 +14,9 @@ export const knownGenesis: KnownGenesis = {
   edgeware: [
     '0x742a2ca70c2fda6cee4f8df98d64c4c670a052d9568058982dad9d5a7a135c5b'
   ],
+  hydradx: [
```

Comment:
Do you have the precious hash as well - accounts tied to the previous hash can also match to the new if both supplied.

Reply:
If this is the current, all ok.